pytest_deadfixtures.py

```
@@ -87,33 +188,29 @@ def get_fixtures(session):
     fm = session._fixturemanager
-    curdir = py.path.local()
+    curdir = Path.cwd()
 
     for fixturedefs in fm._arg2fixturedefs.values():
         assert fixturedefs is not None
         if not fixturedefs:
             continue
         for fixturedef in fixturedefs:
-            loc = getlocation(fixturedef.func, curdir)
+            module = fixturedef.func.__module__
+            if module.startswith("_pytest.") or module.startswith("pytest_"):
```

Ruff (0.15.14)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)